src/workspace-utils.ts

```diff
@@ -19,3 +19,13 @@ export async function isEmberCliProject(): Promise<boolean> {
 
   return emberCliBuildFile.length > 0;
 }
+
+export async function packageAvailable(packageName: string): Promise<boolean> {
+  const files = await workspace.findFiles(
+    `**/node_modules/${packageName}/package.json`,
```

Comment:
It may be incorrect path in case of monorepos, pnpm & such. 
Likely, we need to get information about available ember addons from project (from uELS)
https://github.com/lifeart/ember-language-server/blob/component-context-info-origin/src/project.ts#L40

Reply:
Roger that.